src/router/route.rs

```diff
@@ -17,7 +17,7 @@ pub struct Route<'a, State> {
 }
 
 impl<'a, State: 'static> Route<'a, State> {
-    pub fn new(router: &'a mut Router<State>, path: String) -> Self {
+    pub(crate) fn new(router: &'a mut Router<State>, path: String) -> Self {
```

Comment:
Can I ask why we made this change?

Reply:
IIRC, crate user can not create or get a ref of a router, expose `fn new(...)` here is meanless.

Reply:
thanks